torch/_meta_registrations.py

```diff
@@ -3179,6 +3179,24 @@ def _meta_foreach_inplace(*args, _scalar_op=None, **kwargs):
     return
 
 
+@register_meta([aten._foreach_norm])
+def meta__foreach_norm(self, ord=2):
```

Comment:
Can you implement it as the other foreach ops, by dispatching this metafunction to that of `norm`?

Reply:
Do you mean the decomp? The other meta registrations are similar to this current one

Reply:
The other meta foreach decomps dispatch the bulk of the checks to the non-foreach ops in a for-loop. See for example
https://github.com/pytorch/pytorch/blob/0f2de04d55394ac4bcb36c7cf0cc6eee69630551/torch/_meta_registrations.py#L3093

Reply:
Oh ah I see what you mean now. Yes I had this before and moved it to its own function when the storage_offset shenanigans showed up. Good point to move it back!